### DISCUSSION TITLE
Colors!

### DISCUSSION THREAD

**Discussion by TannerJ07** [TodePond/Arroost#300] [Show and tell]
![Colors](https://github.com/TodePond/Arroost/assets/130108795/19d355b0-55c5-40ec-a5b2-5ec78ebf3169)

**Comment by mahdoosh1** [comment]
show us what it does!

**Reply by TannerJ07** [reply]
Sure, I'll send a short video later. All it does though is alternate through the 8 colors.

**Reply by TannerJ07** [reply]
Or I could forget about it and do it today instead.

**Reply by mahdoosh1** [reply]
be free to share your creations and creativity!

there are so many things possible with this

**Comment by TannerJ07** [comment]
https://github.com/TodePond/Arroost/assets/130108795/f88aa89b-6b76-48a9-a732-4b65fdeac1b6

**Reply by mahdoosh1** [reply]
cool! we can have a party now
